docs/accordproject-faq.md

```diff
@@ -0,0 +1,56 @@
+
+### What benefits do Smart Legal Contracts provide?
+
+Contractual agreements sit at the heart of any organization, governing relationships with employees, shareholders, customers, suppliers, financiers, and more. Yet contracts today are not capable of being managed as the valuable assets they are. Currently, a contract exists as a static text documents stored in cloud storage services, dated contract management systems, or even email inboxes. Often these documents are Word files or PDFs that can only be interpreted by humans. A smart legal contract, by contrast, can be interpreted by machines. 
+
+Smart Legal Contracts can be easily searched, analyzed, queried, and understood. By associating a data model to a contract, it is possible to extract a host of valuable data about a contract or draft a series of contracts from existing data points (i.e., variables and their values).
+
+The data model is used to ensure that all of the necessary data is present in the contract, and that this data is valid. In addition, it provides the necessary structure to enable contracts to "come alive" by adding executable logic.
+
+The result is a contract that is:
+
+
+* Searchable —
+* Analyzable —
+* Real-time —
+* Integrated —
+
+Consequently, contracts are transformed from business liabilities in constant need of management to assets capable of providing real business intelligence and value.
+
+### What is the Accord Project and what is its purpose?
+
+The Accord Project is a non-profit, member-driven organization that builds open source code and documentation for smart legal contracts for use by transactional attorneys, business and finance professionals, and other contract users. Open source means that anyone can use and contribute to the code and documentation and use it in their own software applications and systems free of charge.
+
+The purpose of the Accord Project is to establish and maintain a common and consistent legal and technical foundation for smart legal contracts. The Accord Project is organized into working groups focused on transactions: supply chain, financial services, intellectual property, venture and token sales, real estate and construction; and also on dispute resolution. The transactional working groups are assisted by the Technology Working Group, which builds the underlying open source code and specifications to codify the knowledge of the transactional working groups. More details about the internal governance of the Accord Project are available here.
```

Comment:
Is there a link missing at the end of this paragraph?

Reply:
Maybe remove `focused on transactions: supply chain, financial services, intellectual property, venture and token sales, real estate and construction; and also on dispute resolution.`?

Reply:
Agreed I haven't seen much traction with those workgroups, at least on Slack. It maybe better to say something like join Slack or contact us for whatever your contractual use case is

Reply:
We elected to focus on core tech to begin with. The others will likely come in time.

Reply:
changed